src/assets/icons/index.ts

```diff
@@ -1,5 +1,5 @@
 export { ReactComponent as IcSearch } from './icon_search.svg';
-export { ReactComponent as IcPlus } from './icon_plus.svg';
+export { ReactComponent, default as icPlus } from './icon_plus.svg';
```

Comment:
```suggestion
export { default as icPlus } from './icon_plus.svg';
export { ReactComponent as IcPlus } from './icon_plus.svg';
```
이렇게 하면 된다는 말이었습니다!!! 그리고 이렇게 해도 스타일 깨질텐데 혹시 보드 페이지 yarn start 해서 확인해보시고 커밋하셨나요??

Reply:
네 확인해봤는데 깨지지는 않네요
그래도 언니가 써준 방식이 더 맞는 것 같아서 다시 커밋하겠습니다!

Reply:
두번째로 바꾼건 문제 없었나보네여 근데 문제가 있을 것 같이 생겼는데 혼란이네요 여튼 리뷰 반영 감사합니다!!